### ISSUE TITLE
Modifying the prefab gltf files breaks scene file 

### ISSUE THREAD

**Issue by ethereumdegen** [rewin123/space_editor#218]
If i have a saved scene file and then i need to modify the GLTF files for the prefabs for any reason, this will likely break the scene file... which is extremely unfortunate bc then i lose my ENTIRE level and all that work. 

```
2024-02-25T22:32:03.545644Z ERROR bevy_asset::server: The file at 'models/doodads/tree_purple_1.glb' does not contain the labeled asset 'Mesh0/Primitive1'.
2024-02-25T22:32:03.555626Z ERROR bevy_asset::server: The file at 'models/doodads/tree_pink_1.glb' does not contain the labeled asset 'Mesh0/Primitive1'.
2024-02-25T22:32:03.563857Z ERROR bevy_asset::server: The file at 'models/doodads/birch_yellow.glb' does not contain the labeled asset 'Mesh0/Primitive1'.

```

can there be a way for me to... instead of spawning prefabs..... spawn entities and attach a 'model name' to it and then have a 'model name lookup map' => gltf file  .  Basically I want my scene entities to represent THINGS that have models, not actually BE the models.  So they shouldnt break when the model data is missing or different, maybe it would just show a checkered cube (denoting a missing asset) until i fix the mapping file that maps the model_name => gltf file 

i would , for example, just populate my scene with a bunch of cubes (and name them their type name and use that to load them into my game properly  )   but then i cant easily 'visualize' what my scene will look like that way.   

TLDR Basically i dont like using prefabs i think there should be some 'middleman' option where i can place an entity that , in the editor, visually  LOOKS like a gltf  but isnt literally a prefab .   this is how other game engine scene editors work .

**Comment by ethereumdegen** [comment]
Oh now i see...

the way the GLTF prefab import works is it make an AssetMesh component and points at the mesh like .. 
models/rpg_dagger_03.glb#Primitive0

but what would be nice i if i could instead add a component like  AssetModel  and give it 
models/rpg_dagger_03.glb#Scene0 , or even give it a string which uses a manifest  look-up text file to go from that string to the GLTF file path

**Comment by naomijub** [comment]
Probably can be done by adding it to the editor core/ gltf unpack module, and it can be added to the components. Let's see if @rewin123  has another suggestion

**Comment by rewin123** [comment]
> Oh now i see...
> 
> the way the GLTF prefab import works is it make an AssetMesh component and points at the mesh like .. models/rpg_dagger_03.glb#Primitive0 [and if that is missing, the editor panics ]
> 
> but what would be nice i if i could instead add a component like AssetModel and give it models/rpg_dagger_03.glb#Scene0 , or even give it a string which uses a manifest look-up text file to map => from that string to the GLTF file path . and if for whatever reason that gltf model isnt loading properly, visually show a pink cube or something obviously denoting an error, but gracefully without panic.

You can use **GltfPrefab**, which load full scene without separating gltf. It must solve your problem

**Comment by rewin123** [comment]
Its not a bug or problem which need to solve
